python/console/console_output.py

```diff
@@ -85,6 +81,34 @@ def isatty(self):
         return False
 
 
+FULL_HELP_TEXT = QCoreApplication.translate("PythonConsole", """QGIS Python Console
+======================================
+
+The console is a Python interpreter that allows you to execute python commands
+QGIS libraries are preloaded and can be used directly from the console.
```

Comment:
Maybe like this to make it more clear?

```suggestion
QGIS and Qt modules are already imported and can be used directly.
```

or even
```
Modules from QGIS (analysis, core, gui, 3d) and Qt (QtCore, QtGui, QtNetwork, 
QtWidgets, QtXml) as well as Python's math, os, re and sys modules are already
imported and can be used directly.
```